### PR TITLE
Prepare WPP for production usage

### PR DESCRIPTION
**Pull request by ffiebig**
Prepare WPP for production

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ var webpayPlusMallRouter = require("./routes/webpay_plus_mall");
 var webpayPlusMallDeferredRouter = require("./routes/webpay_plus_mall_deferred");
 
 var app = express();
+if (app.settings.env == "development") {
+  require("dotenv").config();
+}
```
Comment (gdespirito) [inline app.js:16]: Why you only require .env on development?

**Comment by gdespirito** [comment]
I think you need an .env.example on this project, or add the possible values it can hold on Readme file.

File: app.js (hunk 1 above)
Comment (ffiebig) [reply app.js:16]: Because in heroku is easier to load env vars without dot env

**Comment by ffiebig** [comment]
I don't think it's necessary, why would anyone want to put these examples in production mode ?

**Review by gdespirito — APPROVED** [review]

**Pull request merged**